samples/account-linking/csharp/Source/Services/Gmail/GmailServiceClient.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Net.Http.Headers;
+
+namespace Microsoft.Teams.Samples.AccountLinking.SampleClient.Services.Gmail
+{
+    public class GmailServiceClient
+    {
+        private readonly ILogger<GmailServiceClient> _logger;
+
+        private readonly HttpClient _httpClient;
+        public GmailServiceClient(
+            ILogger<GmailServiceClient> logger,
+            HttpClient httpClient)
+        {
+            _logger = logger;
+            _httpClient = httpClient;
+        }
+
+        public async Task<GmailUserProfile> GetGmailUserProfile(string email, string userAccessToken)
+        {
+            try
+            {
+                var request = new HttpRequestMessage
+                {
+                    RequestUri = new Uri($"https://gmail.googleapis.com/gmail/v1/users/{email}/profile"),
```

Comment:
Add link to the Gmail API reference where this is documented

Reply:
https://developers.google.com/gmail/api/reference/rest/v1/users/getProfile